src/policy-store.ts

```diff
@@ -335,4 +340,45 @@ export class PolicyStore extends PolicyStoreBase {
     });
     return policies;
   }
+
+  /**
+   * Takes in an absolute path to a directory containing .cedar files and adds the contents of each
+   * .cedar file as policies to this policy store. Parses the policies with cedar-wasm and, if the policy store has a schema,
+   * performs semantic validation of the policies as well.
+   * @param absolutePath a string representing an absolute path to the directory containing your policies
+   * @returns An array of created Policy constructs.
+   */
+  public addPoliciesFromPath(absolutePath: string): Policy[] {
+    if (!fs.statSync(absolutePath).isDirectory()) {
+      throw new Error(
+        `The path ${absolutePath} does not appear to be a directory`,
+      );
+    }
+    const policyFileNames = fs
+      .readdirSync(absolutePath)
+      .map((f) => path.join(absolutePath, f))
+      .filter((f) => !fs.statSync(f).isDirectory() && f.endsWith('.cedar'));
+
+    if (this.validationSettings.mode === ValidationSettingsMode.STRICT && !this.schema) {
+      throw new Error(
+        'A schema must exist when adding policies to a policy store with strict validation mode.',
+      );
+    }
+
+    if (this.validationSettings.mode === ValidationSettingsMode.STRICT && this.schema?.cedarJson) {
```

Comment:
This line is still not fully covered by unit tests. I don't think it makes sense to test `this.schema?.cedarJson`:

1) We've already test the fact that this.schema should be not null in line 362, when validation settings are STRICT
2) `cedarJson` cannot be null, at least it can be an empty string
3) At that point, if validation settings are STRICT we should proceed, for every policy, with a validation against the schema

Reply:
Without that check we need one of these two: 

1. a non-null assertion
2. nested if statements

I dislike both as code patterns, so that's why I did this. 

However in this case, since the empty string would get around validation, you're raising a valid point that this is essentially a bug. So I'll opt for the nested if statement solution.